### PR TITLE
Add scripts to add a new adapter to latest/stable

### PR DESCRIPTION
**Pull request by AlCalzone**
This adds a new command to get a new adapter into the latest repo:
```
gulp addToLatest --name <adapter-name> --type <adapter-type>
```

Fixes: #480 

/cc @GermanBluefox @Apollon77

### CODE REVIEW

**Comment by Apollon77** [comment]
Is the new entry sorted in alphabetical ? I did not saw code to handle this ...

**Comment by AlCalzone** [comment]
Yes

**Comment by AlCalzone** [comment]
Can't upload the Screenshot here, there's a sortrepo call in scripts.js

**Comment by AlCalzone** [comment]
sure thing

File: lib/scripts.js [hunk 1]
```diff
@@ -371,61 +386,265 @@ function createList(fileName) {
...
+        const file = argv._[argv._.indexOf('list') + 1];
+        if (file) {
+            createList(file);
         } else {
             createList(__dirname + '/../list.html');
         }
+    } else if (argv._.includes('addToStable')) {
+        const {name, version} = argv;
+        if (typeof name !== 'string') fail('Please specify the adapter name!');
+        if (typeof version !== 'string') fail('Please specify the version to add to stable!');
```
Comment (GermanBluefox) [inline lib/scripts.js]: If the version not provided, we can get the latest automatically. The function is here anyway

Comment (AlCalzone) [reply lib/scripts.js]: True. Will update it later

**Comment by GermanBluefox** [comment]
Super! Thanks

**Pull request merged**